src/main/java/ch/heigvd/amt/stack/application/GamificationFacade.java

```diff
@@ -21,8 +30,13 @@
   GamificationRepository gamificationRepository;
   @Inject
   SessionRepository sessionRepository;
+  @Inject
+  CredentialRepository credentialRepository;
+
+  @Getter
+  private int lastRankingPage;
 
-  private CredentialId getCredential(String forTag) {
+  public CredentialId getCredential(String forTag) {
```

Comment:
Please keep this `private` and prefer exposing a different facade method with a dedicated `DTO`.